### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1011] [Galaxy Trivia] [answered]
<img width="270" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/1bc14e38-d682-44d1-b73a-504bf3244f5c">

**Comment by peterbaccaro** [comment, marked as answer]
![image](https://github.com/ParanoidUser/yolo/assets/74666079/c0fff017-f75f-4ac6-82f5-bb6becfba753)
